colossalai/gptq/__init__.py

```diff
@@ -0,0 +1,14 @@
+HAS_AUTO_GPTQ = False
+
+try:
+    import auto_gptq
+    HAS_AUTO_GPTQ = True
+except ImportError:
+    warnings.warn('please install auto-gptq from https://github.com/PanQiWei/AutoGPTQ')
+    HAS_AUTO_GPTQ = False
```

Comment:
Maybe you don't need to check it twice, and you can just import `HAS_AUTO_GPTQ` here.